set.mm

```diff
@@ -243736,6 +243736,13 @@ the additive structure must be abelian (see ~ ringcom ), care must be
       ( ccrg wcel crg ccmn iscrng simprbi ) ADEAFEBGEABCHI $.
   $}
 
+  ${
```

Comment:
TODO: This can also be used in other theorems in Main.  One example is ringnegl.  See my recent contribution #4025 .  One runs through the reference list of ringgrp and looks for applications of this theorem followed by a syl using it as second reference.